app/assets/javascripts/components/students/assign_button.jsx

```diff
@@ -241,7 +241,8 @@ const AssignButton = React.createClass({
       let removeButton;
       let articleLink;
       ass.course_id = this.props.course_id;
-      const article = CourseUtils.articleFromAssignment(ass);
+      const article = CourseUtils.articleFromAssignment(ass, this.props.course.home_wiki);
+      if (!article.formatted_title) { CourseUtils.formattedArticleTitle(article, this.props.course.home_wiki); }
```

Comment:
Is this necessary? Unlike in `assignment.jsx`, this article object is always generated by `CourseUtils.articleFromAssignment`, so it should always have a formatted_title.

Reply:
true!